packages/cloud_firestore/cloud_firestore/example/ios/Podfile

```diff
@@ -52,6 +52,7 @@ target 'Runner' do
   end
 
   pod 'FirebaseFirestore', :git => 'https://github.com/invertase/firestore-ios-sdk-frameworks.git', :tag => "#{firebase_sdk_version}"
+  pod 'nanopb'
```

Comment:
Shouldn't it be added to the firestore podspec?

Reply:
Yes, I wasn't sure if we should add it there as it's only needed if you use the zipped framework. But I think it should because nanopb is a dependency of iOS Firestore so it don't think it'll make a difference if we add to our podspec. Will change 👍